test/integration/datapath/datapath_linux_test.go

```diff
@@ -10,14 +10,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/Azure/azure-container-networking/test/integration"
+	k8s "github.com/Azure/azure-container-networking/test/integration"
```

Comment:
Wouldn't this break stuff ? Given this import was used. If not being used, we can remove it altogther.

Reply:
it's used